### ISSUE TITLE
`iesave`: error when more than one variable is used in `idvars`

### ISSUE THREAD

**Issue by luizaandrade** [worldbank/ietoolkit#347]
If more than one variable is used in `idvars` and one of them has missing values, the code breaks on the following lines:

```
- if _rc {
  - capture assert !missing(`idvars')
  = capture assert !missing(user_uuid session)
  - if _rc {
  - count if missing(`idvars')
  = count if missing(user_uuid session)
```

The error I see is

```
user_uuidsession not found
```

I think this is because `missing()` doesn't take multiple variables as input.

**Comment by kbjarkefur** [comment]
I was able to reproduce this error using this small example:

```
* Example generated by -dataex-. For more info, type help dataex
clear
input float(varA varB) str9 varC
1 1 "some data"
1 2 "some data"
1 . "some data"
1 4 "some data"
1 5 "some data"
2 1 "some data"
2 2 "some data"
2 3 "some data"
. 4 "some data"
2 5 "some data"
end
    
iesave "~/delete-out/issue347.dta", idvars(varA varB) version(15)
```

`idvars()` option should be able to handle multiple variables, so this is a bug.

@luizaandrade , just to make sure that we are on the same page. Missing values in either of these variables should still be an error right? But an error that the command handles and provide useful information about instead of the current cryptic message.

**Comment by kbjarkefur** [comment]
Pushed a fix. It still generates and error, but now, following a non-zero return code from``cap isid `idvars'``, the command tests one idvar at the time. This is the output if two idvars are used and both of them have missing values.

![image](https://github.com/worldbank/ietoolkit/assets/15911801/b883cb1e-185b-457d-bbfc-34ad94c3a1b6)

**Comment by kbjarkefur** [comment]
Do you have time to look at https://github.com/worldbank/ietoolkit/pull/348 before I merge it?

**Comment by luizaandrade** [comment]
That was quick! Yes, I agree it should return an error when these are missing

**Comment by luizaandrade** [comment]
> Do you have time to look at #348 before I merge it?

Yes, I can take a look at it this week

**Comment by kbjarkefur** [comment]
We are about to publish a new version of `ietoolkit`. Did you have time to test this?

**Comment by kbjarkefur** [comment]
merged to `develop` in #348
